dojson/overdo.py

```diff
@@ -63,7 +63,7 @@ class Overdo(object):
     """Translation index."""
 
     def __init__(self, bases=None, entry_point_group=None):
-        """Constructor."""
+        """Initialize."""
```

Comment:
Moderate: I agree that this would be better according to the technical writing training. 

But should we wait on having files changed until the style guidelines have been set in place. Mainly to avoid goes-and-backs with changes in files that might make history and blames more inefficient to track changes.

Reply:
These and the following comment, are not my initiative to apply documentation, but actual errors that had the tests failing.